src/vdaf/prio3.rs

```diff
@@ -554,18 +554,11 @@ where
 
     fn random_size(&self) -> usize {
         if self.typ.joint_rand_len() == 0 {
-            // Two seeds per helper for measurement and proof shares, plus one seed for proving
-            // randomness.
-            (usize::from(self.num_aggregators - 1) * 2 + 1) * SEED_SIZE
+            // One seed per (share, proof) pair
```

Comment:
I think it would be worthwhile to account for how the different seeds are used in this comment, even if we cancel out the -1 and +1 in the formula.
```suggestion
            // One seed per helper (share, proof) pair, plus one seed for proving randomness.
```